syntax = 'proto3';

package shop;

service CartService {
  rpc Cart(CartRequest) returns (CartResponse) {}
}

message CartRequest {
  string message = 1;
}

message CartResponse {
  string message = 1;
}